scripts/src/main/resources/scripts/migration

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/functions
+
+doEcho "\nRun migration scripts to version ${1}."
+migrations="$(dirname "${0}")/migrations/*"
+for migration in $(ls ${migrations} | sort -n)
+do
+  from_version="${1}"
+  to_version="${migration/*\//}"
```

Comment:
could be simplified, if you just `cd` to the migrations folder (first check if the folder exists and otherwise log a warning what will also help if something goes wrong) and then simply use globbing with `sort -n`. Then you will only get the plain version and do not have to extract the filename from the entire path.
However, just a nice-to-have comment - nothing wrong with your code.